src/modules/Checklist copy/ChecklistV2.tsx

```diff
@@ -218,12 +218,12 @@ const ChecklistV2 = ({
                                     )
                                 }
                                 deleteAttachment={(
-                                    attachmentId: number
+                                    attachmentId: number | string
                                 ): Promise<void> =>
                                     api.deleteChecklistAttachment(
                                         plantId,
                                         checklistId,
-                                        attachmentId
+                                        attachmentId as number
```

Comment:
i stedet for as number kan du vel bare parseInt(attachmentId) liker ikke å bruke "as Type"

Reply:
Får typerror på parseInt